api/v1alpha1/appgroup_types.go

```diff
@@ -262,10 +262,16 @@ func (in *ApplicationGroup) Progressing() {
 // RollingBack sets the meta.ReadyCondition to 'True' and
 // meta.RollingBack reason and message
 func (in *ApplicationGroup) RollingBack() {
```

Comment:
Just to clarify, what is the difference now between these two conditions?

Reply:
RollingBack is on upgrade failures where we rollback to previous successful spec and Reversing is for installation failures
Not sure if that was the question 🤔

Reply:
I see, is there a reason we have a distinction between these two? Aren't they essentially the same action?

Reply:
My personal opinion is that we just use reversing reason as universal for both rollback and for delete

Reply:
Hmm - we do some special handling for RollingBack early in the reconciliation - like restoring the lastSuccessfulSpec and re-applying the Workflow generated from it. 
So we need to handle the events separately. Does that make sense?

Reply:
I see, I still think reversing should be moved from the ReadyCondition to DeployCondition

Reply:
This may need to be a longer conversation about what each condition should be capturing